src/afpusers/files/afpusers.py

```diff
@@ -44,7 +44,8 @@ def AFPUsers():
 
     for line in p.stdout:
         line = line.rstrip()
-        if AFPD_PROCESS in line:
+        if bytes(AFPD_PROCESS, encoding="ascii") in line:
+            line = line.decode('ascii')
```

Comment:
Can we remove all of these bytes() and use encoding='utf8' in the subprocess.Popen call instead?

Or use `line = line.decode().rstrip()`

Reply:
Heinrich, Did you plan to make that suggested change so we can merge this in?